tests/src/test/java/io/streamnative/pulsar/handlers/kop/BasicEndToEndTestBase.java

```diff
@@ -331,4 +334,39 @@ protected void kafkaConsumeCommitMessage(KConsumer kConsumer,
         }
     }
 
+
+    @Test(timeOut = 20000)
+    public void testDeletePartition() throws Exception {
```

Comment:
It's better to move `testDeletePartition` into `BasicEndToEndKafkaTest`. The `XXXBase` class should not contain any test method.

Reply:
initially I have written the test here because BasicEndToEndKafkaTest is "specialised" for entryFormat=kafka.

Also I have another test in the Proxy that extends this class and so I automatically inherited this method.

not a big deal.
I have moved the test to BasicEndToEndKafkaTest